meetings/2025-03-05.md

```diff
@@ -0,0 +1,121 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2025-03-05T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Agenda
+
+
+## Present
+* [Michiel de Jong](https://michielbdejong.com)
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
+* Rahul Gupta
+* Hadrian Zbarcea
+* Jesse Wright
+* Rui Zhao
+* Theo
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+
+* elf Pavlik
+
+## Topics
+
+### PRACTITIONERS MEETUP ON TRANSPORTATION
+* JZ: Thursday March 6, 15-17:00 UTC https://meet.jit.si/solid_practioners will focus on Solid and Transportation with talks from Pietr Colpaert and four others working on decentralized transportation.  All invited.
+
+### Connecting a Wallet with a Pod
+* MdJ: Inrupt, KIT and SURF are all experimenting with the connection between pods and wallets. Let's compare notes! https://docs.inrupt.com/data-wallet/patterns-and-flows/
+* MdJ: We are doing it with the Dutch Univeristy. We try to share VCs from my wallet. Instead of it being self contained it refers to a document on the pod. The question is how the signature works then. Does the document contain signature. How does it work with the Inrupt wallet. Can you share a document in the same way as you would share a VC?
```

Comment:
```suggestion
* MdJ: Inrupt, KIT, and SURF are all experimenting with the connection between pods and wallets. Let's compare notes! https://docs.inrupt.com/data-wallet/patterns-and-flows/
* MdJ: We are doing it with the Dutch Univeristy. We try to share VCs from my wallet. Instead of it being self contained, it refers to a document on the pod. The question is,  how does the signature work then? Does the document contain signature? How does it work with the Inrupt wallet? Can you share a document in the same way as you would share a VC?
```